### PR TITLE
Making questions better

### PR DESCRIPTION
**Pull request by AlexanderLea**

### CODE REVIEW

File: Question1.cs [hunk 1]
```diff
@@ -56,16 +80,16 @@ public async Task<PersonModel> GetPerson(string personName)
...
             };
 
-            return personModel;           
+            return personModel;
         }
-    
+
         public async Task<PersonModel> GetLastRecordEditor(string personName)
         {
             var personRepository = _container.Resolve<IPersonRepository>();
```
Comment (BradleySharp) [inline Question1.cs:88]: I think it might be worth mixing naming string/var/int and maybe having one wrong

File: Question2.cs [hunk 2]
```diff
@@ -14,20 +14,16 @@ public class Question2
...
             SqlConnection connection = new SqlConnection(connectionString);
-            var command = new SqlCommand(queryString, connection);
-            command.Parameters.AddWithValue("@age", paramValue);
+            
+            var queryString = "SELECT ItemId, FirstName, LastName "
+                + "FROM People "
+                + $"WHERE JobTitle = '{jobTitle}'"
+                + "ORDER BY Age DESC;";
+
+            var command = new SqlCommand(queryString, connection);
```
Comment (BradleySharp) [inline Question2.cs]: var command = new SqlCommand($"SELECT ItemId, FirstName, LastName FROM People WHERE JobTitle = '{jobTitle}' ORDER BY Age DESC;", connection);

Comment (dgolovniov) [reply Question2.cs]: Not sure if this was intentional, but there will be no space between job title and the Order by statement. :) which can be considered a check for being attentive.

Comment (AlexanderLea) [reply Question2.cs]: Certainly not deliberate :)

**Pull request merged**